README.md

```diff
@@ -1 +1,26 @@
-# library
\ No newline at end of file
+# library
+
+This project entails building a simple libary where a user can create a new book, put it into the library, change the status i.e, read, or not yet read, and even remove the book from the library.
+
+## To get started
+
+- Clone the repository and cd into the library project.
+- ` $ git clone https://github.com/karvel-code/library.git`
+- ` $ cd library`
+
```

Comment:
- [ ] Kindly be more specific on how to run the project in the browser here, explain to the user what they should do after the `cd` into `library` to interact with your application. You can use this [template](https://github.com/microverseinc/readme-template) to structure your README so that it stands out.